### DISCUSSION TITLE
how can ı remove playback speed widget

### DISCUSSION THREAD

**Discussion by sahinlokman** [tomasklaen/uosc#843] [Q&A]
![image](https://github.com/tomasklaen/uosc/assets/154338591/20459c4a-d450-44b8-a68a-90d136e918ea)
ı want to remove this widget.

**Comment by OmiChrome** [comment]
you can edit `uosc.conf ` more info https://github.com/tomasklaen/uosc?tab=readme-ov-file#options
as  https://github.com/tomasklaen/uosc/blob/dc73278d657eec1b7236a4e7ad8466b6fd41f887/src/uosc.conf#L81
remove `speed` from it 

make it as
```
controls=menu,gap,subtitles,<has_many_audio>audio,<has_many_video>video,<has_many_edition>editions,<stream>stream-quality,gap,space,space,shuffle,loop-playlist,loop-file,gap,prev,items,next,gap,fullscreen
```

**Reply by sahinlokman** [reply]
ıt work thank you for your help.
